lib/action_policy/lookup_chain.rb

```diff
@@ -53,21 +66,18 @@ def call(record, **opts)
 
       private
 
-      def lookup_within_namespace(policy_name, namespace)
-        return unless namespace
-        NamespaceCache.fetch(namespace.name, policy_name) do
+      def lookup(policy_name, namespace, strict: false)
```

Comment:
I think, we should keep the old name. This method deals with namespaces, it's not a general lookup.